src/zsk4gm/process_text.py

```diff
@@ -0,0 +1,51 @@
+import string
+import logging
+from collections import Counter
+import os
+
+root_dir = os.path.dirname(os.path.abspath(__file__))  # This gets the directory of the current script
+log_file_path = os.path.join(root_dir, 'logs', 'logfile.log')
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s: %(message)s', filename=log_file_path)
+
+
+def clean_text(text):
+	assert isinstance(text, str)
+	logging.debug(f'Cleaning text: {text}'), 'Input must be a string' 
+
+	text = text.lower()
+	cleaned_text = text.translate(str.maketrans('', '', string.punctuation))
+	
+	assert isinstance(cleaned_text, str), 'Output must be a string'
+	logging.debug(f'Cleaned text: {cleaned_text}')
+	
+	return cleaned_text or ""
+
+def tokenize(text):
```

Comment:
you could still add comments, and in some organizations it is required.  But for class I wanted to stress that if you're going to do one thing or another, I'd do asserts, because they don't get out of date.  So just pointing that out.  Also there are many types of ways to do the comments and they also vary by organization.  Just wanted to mention that for completeness, however your code is very clear and the spacing makes it easy to read.